app/models/user.rb

```diff
@@ -221,7 +222,7 @@ def confirmed_user_has_fields
   end
 
   def jurisdiction_must_belong_to_correct_roles
-    if jurisdictions.any? && !review_staff?
+    if jurisdictions.any? && !review_staff? && !technical_support?
```

Comment:
What I would do is add a new computed value jurisdiction_staff?
check everywhere review_staff? is used, and determine if it should be replaced with jurisdiction_staff?